authentication.py

```diff
@@ -0,0 +1,92 @@
+import re
+from passlib.context import CryptContext
+from fastapi import HTTPException, status
+import jwt
+
+from models import User
+from config import get_settings
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+
+def get_hashed_password(password):
+    return pwd_context.hash(password)
+
+
+async def very_token(token: str):
```

Comment:
**Security Issue**: The function 'very_token' appears to be a typo and should be 'verify_token'. This typo could lead to runtime errors if other parts of the codebase attempt to call 'verify_token' as expected. <br> **Fix**: Rename the function to 'verify_token' to match the expected functionality and avoid potential runtime errors. <br> **Code Suggestion**: 
 ```
 @@ -16,2 +16,2 @@
 -async def very_token(token: str):
 +async def verify_token(token: str):
 ```